phpstan.neon

```diff
@@ -8,6 +8,7 @@ parameters:
     checkFunctionNameCase: true
     checkInternalClassCaseSensitivity: true
     treatPhpDocTypesAsCertain: false
+    reportUnmatchedIgnoredErrors: false
```

Comment:
We should remove this later once this PR is ready to merge and clean up the baseline instead.